scheduleit/config/database.php

```diff
@@ -733,6 +733,168 @@ public function addMeeting($user_id, $meeting)
 
         return $result;
     }
+
+    /**
+     * Get list of onids that have not registered 
+     * for an event.
+     *
+     * @param object $eventId
+     * @return string array
+     */
+     public function getNotRegistered($eventId)
+     {
+      
+       $not_reg_query = "
+      
+       SELECT user_onid FROM meb_invites
+       WHERE fk_event_id = ? AND user_onid NOT IN
+         (SELECT DISTINCT user_onid FROM meb_invites
+         INNER JOIN meb_user ON meb_user.onid = meb_invites.user_onid
+         INNER JOIN meb_booking ON meb_booking.fk_user_id = meb_user.id
+         INNER JOIN meb_timeslot ON meb_booking.fk_timeslot_id = meb_timeslot.id
+         INNER JOIN meb_event ON meb_event.id = meb_timeslot.fk_event_id
+         where meb_timeslot.fk_event_id = ?)
+       ";
+
+         $getList = $this->database->prepare($not_reg_query);
+      
+         $getList->bind_param("ii", $eventId, $eventId);
+         $getList->execute();
+
+         $list = $getList->get_result();
+
+         $listArray = $list->fetch_all(MYSQLI_ASSOC);
+           
+         $getList->close();
+         $list->free();
+         return $listArray;
+    }
+
+     /**
+     * add onid to list of inivted onids for an event
+     * 
+     *
+     * @param string $onid, int $eventId
+     * @return none
+     */
+     public function insertInviteList($onid, $eventId)
+     {
+       $insert_meb_invites_query = "
+         INSERT `meb_invites` (fk_event_id, user_onid)
+         VALUES (?,?);
+       ";
+      
+       $insert = $this->database->prepare($insert_meb_invites_query);
+
+       $insert->bind_param("is", $eventId, $onid);
+       $insert->execute();
+
+       $insert->close();
+    } 
+    
+    /**
+    * delete a booking  
+    * 
+    *
+    * @param string $startTime, int $eventId, string $onid
+    * @return int result
+    */
+    public function deleteBooking($startTime, $eventId, $onid)
```

Comment:
I've also added a `deleteBooking` in my PR #44 that calls the stored procedure that also updates timeslots when bookings are deleted.

Reply:
i'm removing my deleteBooking function on and changing getSlotId to get slotHash and then calling your delete booking.  i'm also deleting the update slots